README.md

```diff
@@ -73,12 +73,22 @@ To run Zilliqa, we recommend the following minimum system requirements:
         libssl-dev libleveldb-dev libjsoncpp-dev libsnappy-dev cmake libmicrohttpd-dev \
         libjsonrpccpp-dev build-essential pkg-config libevent-dev libminiupnpc-dev \
         libprotobuf-dev protobuf-compiler libcurl4-openssl-dev
+    
+    # Install openssl-1.1.1 on 16.04
```

Comment:
Historically we haven't copied over some of our installation code (e.g., mongodb) into the readme, apart from the apt-get install and the brew install.  It might become too unwieldy I think.

Reply:
@Gnnng will refactor this process into a cmake-based one.